packages/grid/x-data-grid-premium/src/tests/exportExcel.DataGridPremium.test.tsx

```diff
@@ -56,12 +56,15 @@ describe('<DataGridPremium /> - Export Excel', () => {
       expect(await act(() => apiRef.current.getDataAsExcel())).not.to.equal(null);
     });
 
-    it('should display export option', () => {
+    it('should display export option', async () => {
       render(<TestCaseExcelExport components={{ Toolbar: GridToolbar }} />);
 
       fireEvent.click(screen.getByRole('button', { name: 'Export' }));
-      expect(screen.queryByRole('menu')).not.to.equal(null);
-      expect(screen.queryByRole('menuitem', { name: 'Download as Excel' })).not.to.equal(null);
+
+      await waitFor(() => {
```

Comment:
Nitpick, it's recommended to usually [have just one assertion](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#having-multiple-assertions-in-a-single-waitfor-callback) in a `waitFor` and leave any follow-up assertions after `waitFor`. 😉 

P.S. In this case, we could probably avoid the `waitFor` [by utilizing `await findByRole`](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-waitfor-to-wait-for-elements-that-can-be-queried-with-find) 👌 🙈 
This would apply to a lot of other cases as well. Forgot about this preference/suggestion when reviewing the other PR. 🙈